src/components/AddProblemToSection/AddProblemToSection.tsx

```diff
@@ -14,6 +15,10 @@ const StyledModalHeader = styled(ModalHeader)`
     padding-bottom: ${gapM};
 `;
 
+const StyledButton = styled(Button)`
+    margin: ${gapM} 0;
```

Comment:
It's not obvious why you need this style here. U have 3 buttons in a row, why not to add margin for container of them?